plugins/bulk-import/src/utils/repository-utils.tsx

```diff
@@ -14,10 +14,13 @@ import {
 export const getRepositoryStatus = (
   status: string,
   isItemSelected?: boolean,
+  isDrawer: boolean = false,
+  selectedRepositories?: number,
 ) => {
   if (isItemSelected) {
     return (
       <span>
+        {selectedRepositories || ''}
         <ReadyIcon
           color="success"
           style={{ verticalAlign: 'bottom', paddingTop: '7px' }}
```

Comment:
Can you update this to

```suggestion
          style={{ verticalAlign: 'sub', paddingTop: '7px' }}
```